scripts/pocket-change.js

```diff
@@ -67,6 +72,25 @@ export default class PocketChange {
     return cr >= lowerCr && cr <= upperCr;
   }
 
+  _treasureNoChallengeRating(actor) {
```

Comment:
Just curious, are these dice rolls based on anything?

Reply:
I just have a need to generate a random number of coins independent of NPC rating for it would take something truly random, but I wasn't sure how to proceed.... perhaps a dynamic dialogue like this https://github.com/unsoluble/fvtt-macros/blob/master/loot-swapping/set-currency.js might be a better solution

Reply:
Hm, yeah I like that as a future enhancement. Will resolve this for now